src/config_dynamic.cpp

```diff
@@ -357,6 +358,9 @@ static AgentTypeVector parse_agents_configuration(const Node& node) {
 		const Node& follow_rate = child["rates"]["follow"];
 		const Node& tweet_rate = child["rates"]["tweet"];
 
+        //Parsing Susceptibility
```

Comment:
This comment is a bit lacking in oomph. The line it comments already has the words 'parse' and 'susceptibility'. I would remove it.